File: sdks/go/pkg/models/shared/metadataqueryindices.go
package shared

// MetadataQueryIndices
// A collection of metadata query indices.
type MetadataQueryIndices struct {
	Entries    []MetadataQueryIndex `json:"entries,omitempty"`
	Limit      *int64               `json:"limit,omitempty"`
	NextMarker *string              `json:"next_marker,omitempty"`
}

File: ts-sdk/src/sdk/models/shared/metadataqueryindices.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { MetadataQueryIndex } from "./metadataqueryindex";



// MetadataQueryIndices
/** 
 * A collection of metadata query indices.
**/
export class MetadataQueryIndices extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=entries", elemType: MetadataQueryIndex })
  entries?: MetadataQueryIndex[];

  @SpeakeasyMetadata({ data: "json, name=limit" })
  limit?: number;

  @SpeakeasyMetadata({ data: "json, name=next_marker" })
  nextMarker?: string;
}
